importmail.py

```diff
@@ -0,0 +1,121 @@
+import email
+import imaplib
+import os
+from datetime import datetime, timedelta
+import time
+from process import time_delta
+import sys
+import argparse
+
+##############################################################################################
+# importmail.py is responsible for extracting and downloading united hatzala email attachments
+# Requirements: Setting an environment variable named MAILPASS containing the password
+#               And another named MAILUSER containing the mail account username
+# Note: 1. This script is being called by united.py prior to DB import
+#       2. If 'detach_dir' is empty the script would download all available files
+#          If it's not, only the last file would be downloaded
+#       3. Consider emptying the mail directory from time to time to speed things up
+#       4. Command line arguments:
+#           --username can be used to set mail user without env var
+#           --password can be used to set mail password without env var
+#           --lastmail is currently set to default True
+##############################################################################################
+
+
+def main():
+    maildir = 'united-hatzala/data'
+    detach_dir = 'static/data/united'
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--username', default='')
+    parser.add_argument('--password', default='')
+    parser.add_argument('--lastmail', action='store_true', default=False)
+    args = parser.parse_args()
+
+    try:
+        if args.username:
+            username = args.username
+        else:
+            username = os.environ['MAILUSER']     # Set and environment variable MAILUSER with the password
+    except:
+        print "Please set env var MAILUSER, or provide one using importmail.py --username <username>"
```

Comment:
But actually, the command line arguments work if you ran `united.py` and not `importmail.py` directly.
You should use `argv[0]` rather than "importmail.py" in the message.